web_src/css/shared/flex-list.css

```diff
@@ -80,6 +80,7 @@
 .flex-item .flex-item-body a {
   color: inherit;
   overflow-wrap: anywhere;
+  display: inline-flex;
```

Comment:
What's this used for? I guess we shouldn't covert everything to flex aggressively, especially for a general tag `a`.

Reply:
Also, even more broken on org PR:

<img width="233" alt="Screenshot 2024-04-15 at 20 59 48" src="https://github.com/go-gitea/gitea/assets/115237/01e226fa-5641-4d3a-a157-73dcfb815b1b">

Reply:
Problem is the inner `.truncated-name` is `display: inline-block` and that causes the expansion. If I remove it, truncation won't work. If I set `display: block` on it, the item will render on multiple lines for org PR.

Reply:
Found the fix with `vertical-align` in https://github.com/go-gitea/gitea/pull/30345/commits/db2043d1f5566220747748e5426f74541fbfce86:

<img width="491" alt="Screenshot 2024-04-15 at 21 06 13" src="https://github.com/go-gitea/gitea/assets/115237/8b381469-d05a-47c6-a4e9-bec04b23825f">